man/org.freedesktop.login1.xml

```diff
@@ -102,12 +102,26 @@ node /org/freedesktop/login1 {
                    in  b interactive);
       FlushDevices(in  b interactive);
       PowerOff(in  b interactive);
+      PowerOff1(in  b interactive
```

Comment:
Please use more descriptive names. Also, there's no need to make it optional - it's a new interface, so it can always be the default.

Something like PowerOffWithInhibitors or such.

Reply:
@bluca thanks for the review. What do you mean by make it optional? Both old and new method should exist, old one for backward compatibility.

Reply:
I mean the boolean parameter - there's no need for that. Just make it always do it when the new methods are used.

Reply:
It make sense. Thanks.